tests/.eslintrc.js

```diff
@@ -1,5 +0,0 @@
-module.exports = {
```

Comment:
Any particular reason this config is deleted?

Reply:
It's config for a now empty folder so not needed any more.